Misc/NEWS.d/next/Library/2023-02-22-02-14-38.gh-issue-81371.tUo0u1.rst

```diff
@@ -0,0 +1 @@
+This PR may fix the issue 81371
```

Comment:
May? Or does it? Please, also specify what the issue is. These are user-facing notes. It must be clear without a github link.

Reply:
> May? Or does it? Please, also specify what the issue is. These are user-facing notes. It must be clear without a github link.

I'm not 100% sure if it does. In the original issue those methods just returns some awaitable objects that doesn't pass the check. And I fixed it, but only for this certain example (I will add the tests soon). The obstacle is that I don't know if there are similar functions somewhere in the inerpreter. I'm not an internals expert. It would be great if you or someone else could suggest me such places.